php.ini-development

```diff
@@ -148,8 +148,9 @@
 ;   Development Value: 5
 ;   Production Value: 5
 
+; This directive is DEPRECATED.
 ; short_open_tag
-;   Default Value: On
+;   Default Value: Off
```

Comment:
As the defaults all match now, the short_open_tag entry should be dropped from here.

Reply:
Does this need to be done with the php.ini-production file too?

Reply:
Yes